vault/resource_consul_secret_backend_test.go

```diff
@@ -74,6 +77,7 @@ func TestConsulSecretBackend(t *testing.T) {
 					resource.TestCheckResourceAttr("vault_consul_secret_backend.test", "address", "consul.domain.tld:8501"),
 					resource.TestCheckResourceAttr("vault_consul_secret_backend.test", "token", token),
 					resource.TestCheckResourceAttr("vault_consul_secret_backend.test", "scheme", "https"),
+					resource.TestCheckResourceAttr("vault_consul_secret_backend.test", "local", "false"),
```

Comment:
Please could you add a case where we expect it to be true as well?

Reply:
✅ Added

```
=== RUN   TestConsulSecretBackendRole
--- PASS: TestConsulSecretBackendRole (0.35s)
=== RUN   TestConsulSecretBackendRoleNameFromPath
--- PASS: TestConsulSecretBackendRoleNameFromPath (0.00s)
=== RUN   TestConsulSecretBackendRoleBackendFromPath
--- PASS: TestConsulSecretBackendRoleBackendFromPath (0.00s)
=== RUN   TestConsulSecretBackend
--- PASS: TestConsulSecretBackend (0.66s)
PASS
ok      github.com/hashicorp/terraform-provider-vault/vault     1.369s
```